pdd

```diff
@@ -422,6 +422,8 @@ def get_parser():
            help='quiet mode for background timer/stopwatch')
     addarg('keywords', nargs='*', help='diff/add/subtract from today or now')
 
+    addarg('-v', '--version', action='version', version=f"%(prog)s version {_VERSION_}")
```

Comment:
Please add to the README and completion scripts as well.

Reply:
Sorry that it wasn't complete! I pushed the changes. I hope they are included in the original PR now.